crypto/stack/stack.c

```diff
@@ -250,7 +250,7 @@ void OPENSSL_sk_pop_free(OPENSSL_STACK *st, OPENSSL_sk_freefunc func)
         return;
     for (i = 0; i < st->num; i++)
         if (st->data[i] != NULL)
-            func(st->data[i]);
+            func((char *)st->data[i]);
```

Comment:
well, yeah.  but it's been that way ever since safestack was created, and probably even longer.  if you have a way to address that in time for 1.1.0, great.